doc/_admin-guide/110_Template_and_rewrite/000_Customize_message_format/001_Templates_and_macros.md

```diff
@@ -52,18 +52,18 @@ destination d_file {
 };
 ```
 
-Macros can be included by prefixing the macro name with a \$ sign, just
+Macros can be included by prefixing the macro name with a $ sign, just
 like in Bourne compatible shells. Although using braces around macro
-names is not mandatory, and the \"\$MESSAGE\" and \"\${MESSAGE}\"
-formats are equivalent, using the \"\${MESSAGE}\" format is recommended
+names is not mandatory, and the \"${MESSAGE}\" and \"${MESSAGE}\"
+formats are equivalent, using the \"${MESSAGE}\" format is recommended
```

Comment:
i guess the original version suggested here the usage of \$ MESSAGE format, but now we are using \$\{MESSAGE\} everywhere, so as we suggest now the "new" format, probably the clarity reason is not relevant anymore?!